### PR TITLE
Add alias "ht" to /om Hidden Type

### PR DESCRIPTION
**Pull request by shanghaii**

### CODE REVIEW

File: config/commands.js [hunk 1]
```diff
@@ -1531,7 +1531,7 @@ var commands = exports.commands = {
 			matched = true;
 			buffer += "- <a href=\"https://www.smogon.com/forums/threads/3521887/\">Classic Hackmons</a><br />";
 		}
-		if (target === 'all' || target === 'hiddentype') {
+		if (target === 'all' || target === 'hiddentype'|| target === 'ht') {
```
Comment (RhythmsAreFromVibrations) [inline config/commands.js]: `'hiddentype'|| target`

there needs to be a space after `'hiddentype'`

**Comment by Zarel** [comment]
Please actually test before making a pullreq.

**Comment by Joimer** [comment]
@shanghaii you could fix this PR by adding a new commit that fixes the space, rebasing these two commits and melding it into one with squash and updating the branch on your fork. But since it's a pretty acceptable and tiny changed, I've done it myself here: ed8ec7d49913eecffbef7eb425ba3d2b78e207c7

You can test your code changes with npm test, using it in the directory of the repository.

**Comment by shanghaii** [comment]
Noted.

**Pull request closed without merging**